libc/Makefile

```diff
@@ -15,7 +15,7 @@ export AR := $(PREFIX)ar
 export AS := $(PREFIX)as
 endif
 
-SHELL := /bin/bash -o pipefail
+SHELL := bash -o pipefail
```

Comment:
A word of caution, this assumes that bash is on the PATH. Reasonably safe assumption, but still worth being aware of.

Reply:
imho it's as fragile to depend on bash being on the PATH as it is to assume that it's available as `/bin/bash`

Reply:
The makefile already assumes that other tools like `rm` are on the path. These are likely to be installed in /bin, implying that this change is extremely unlikely to regress behavior. 

Even ignoring that, I don't think I've ever run into a realistic build environment where bash is installed but not on the path. On the other hand, there are many reasons why it wouldn't be in /bin (NixOS, but also for example if it was installed in /usr/local).